library/core/src/std.fe

```diff
@@ -0,0 +1,2 @@
+// TODO: Why is this not needed?
+//use codec::{ Cursor, Encode, Decode }
```

Comment:
Question 1:
Why is this not needed? I can call everything from the std ingot even with an empty file. So what should be defined on this file?